src/gateways/googleAnalyticsClient.ts

```diff
@@ -182,6 +181,10 @@ class GoogleAnalyticsClient {
     this.flushPendingCommands();
   }
 
+  private trackingIsDisabled(): boolean {
```

Comment:
I just realised that Tom might have in mind to make this a normal function in `helpers/analytics.ts`  since he mentioned "other analytics systems"

Reply:
But the cookie in `helpers/analyticsEvents/constants.ts` should stay, right? and just make `trackingIsDisabled` as a exported function in `helpers/analytics.ts`.

Reply:
I think in this case we should move it to the `helpers/contants.ts`